pkg/apis/helm.fluxcd.io/v1/types.go

```diff
@@ -9,6 +9,7 @@ import (
 	"github.com/ghodss/yaml"
 
 	corev1 "k8s.io/api/core/v1"
+	v1 "k8s.io/api/core/v1"
```

Comment:
These are duplicates (`corev1` should be kept).

Reply:
Changed